src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -824,7 +826,7 @@ private async Task<List<LaunchCommand>> GetInitializeCommands()
                         if (!string.IsNullOrWhiteSpace(destination))
                         {
                             commands.Add(new LaunchCommand("-target-select remote " + destination, string.Format(CultureInfo.CurrentCulture, ResourceStrings.ConnectingMessage, destination)));
-
+                            commands.AddRange(_launchOptions.PostRemoteConnectCommands);
```

Comment:
Looks like you have attach and local launch covered, but aren't you missing a case, the non-local (e.g. pipe) launch options?

Reply:
I think you're right, I relocated a commands.AddRange call to correct this. Let me know if that doesn't fix what you're referring to.